src/sc_client/core/sc_client_.py

```diff
@@ -0,0 +1,193 @@
+from __future__ import annotations
+
+import asyncio
+import logging
+import time
+from typing import Callable
+
+import nest_asyncio
+
+from sc_client import ScEventType
+from sc_client.constants import sc_types
+from sc_client.core.async_sc_client_ import AsyncScClient
+from sc_client.models import (
+    AsyncScEvent,
+    AsyncScEventParams,
+    ScAddr,
+    ScConstruction,
+    ScEvent,
+    ScEventParams,
+    ScIdtfResolveParams,
+    ScLinkContent,
+    ScLinkContentData,
+    SCsText,
+    ScTemplate,
+    ScTemplateParams,
+    ScTemplateResult,
+    ScType,
+)
+
+
+# noinspection PyProtectedMember
+# pylint: disable=protected-access
+# pylint: disable=too-many-public-methods
+class ScClient:
```

Comment:
Can we make compatibility with old API?

Reply:
They are similar. We have classes now instead functions. It's easy to migrate